12/umbraco-cms/extending/backoffice-tours.md

```diff
@@ -90,7 +90,7 @@ Below is an explanation of each of the properties on the tour configuration obje
 
     ![Content Type specific tours](<../../../10/umbraco-cms/extending/images/contentTypespecific (1) (1) (1) (1) (1).png>)
 
-    In the image above, two tours are avaibable on the _Welcome_ node:
+    In the image above, two tours are available on the _Welcome_ node:
```

Comment:
to keep the docs consistent, can you please make this change to v10 and v11 as well?